WcaOnRails/app/views/delegates_panel/pending_claims_for_subordinate_delegates.html.erb

```diff
@@ -3,32 +3,28 @@
 <div class="container">
   <h2>Pending WCA ID claims for Delegates managed by <%= @user.name %></h2>
 
-  <% if @user.subordinate_delegates.empty? %>
-    <div class="well">No subordinate Delegates to display</div>
-  <% else %>
-    <div class="panel-group">
-      <% @user.subordinate_delegates.sort_by { |u| u.confirmed_users_claiming_wca_id.size }.reverse_each do |delegate| %>
-        <% users = delegate.confirmed_users_claiming_wca_id %>
-        <div class="panel panel-default">
-          <div class="panel-heading heading-as-link" data-toggle="collapse" data-target="#panel-<%= delegate.id %>">
-            <%= delegate.name %>
-            <span class="badge"><%= users.size %></span>
-          </div>
-          <div id="panel-<%= delegate.id %>" class="collapse panel-collapse">
-            <ul class="list-group">
-              <% if users.empty? %>
-                <li class="list-group-item">No pending claims.</li>
-              <% else %>
-                <% users.each do |u| %>
-                  <%= link_to(edit_user_path(u.id, anchor: "wca_id"), class: "list-group-item") do %>
-                    <%= u.name %> has claimed WCA ID <%= u.unconfirmed_wca_id %>
-                  <% end %>
+  <div class="panel-group">
+    <% @subordinate_delegates.sort_by { |u| u.confirmed_users_claiming_wca_id.size }.reverse_each do |delegate| %>
+      <% users = delegate.confirmed_users_claiming_wca_id %>
+      <div class="panel panel-default">
+        <div class="panel-heading heading-as-link" data-toggle="collapse" data-target="#panel-<%= delegate.id %>">
+          <%= delegate.name %>
+          <span class="badge"><%= users.size %></span>
+        </div>
+        <div id="panel-<%= delegate.id %>" class="collapse panel-collapse">
+          <ul class="list-group">
+            <% if users.empty? %>
+              <li class="list-group-item">No pending claims.</li>
+            <% else %>
+              <% users.each do |u| %>
+                <%= link_to(edit_user_path(u.id, anchor: "wca_id"), class: "list-group-item") do %>
+                  <%= u.name %> has claimed WCA ID <%= u.unconfirmed_wca_id %>
                 <% end %>
               <% end %>
-            </ul>
-          </div>
+            <% end %>
+          </ul>
         </div>
-      <% end %>
-    </div>
-  <% end %>
+      </div>
+    <% end %>
+  </div>
```

Comment:
weird-looking diff, it's just a re-indentation of the code now that it's guaranteed that `@subordinate_delegates` contains at least one user.

Reply:
<3 Github's "Hide whitespace changes" option:

![image](https://user-images.githubusercontent.com/277474/51267268-2c32be80-1972-11e9-8dea-f063d92cf0f9.png)